docs/components/AppLayout.js

```diff
@@ -22,8 +22,19 @@ export default function AppLayout({ children }: Props): Node {
         <Divider />
 
         <Box width="100%" minWidth={0}>
-          <Box padding={4} mdPadding={6} lgPadding={8} width="100%" role="main">
-            {children}
+          <Box
+            padding={4}
+            mdPadding={6}
+            lgPadding={8}
+            marginBottom={12}
+            width="100%"
+            role="main"
+            lgDisplay="flex"
+            justifyContent="center"
+          >
+            <Box width="100%" maxWidth={1440}>
```

Comment:
Can I ask where 1440 comes from?

Reply:
You certainly can and should! I think I was attempting to reverse-engineer the max-size of the current content width, but it seems pretty obvious that I got the math wrong. I think we can change the max-width to 1544 - would that make sense to you?

Reply:
Hmm I'm getting 1552, 1312 for the main column + max 240 for the right-side nav?